R/encoloc.R

```diff
@@ -109,13 +110,13 @@ coloc_wrapper <- function(xqtl_file, gwas_files,
     combined_gwas_lbf_matrix <- bind_rows(gwas_lbf_matrices) %>%
                                 mutate(across(everything(), ~replace_na(., 0)))
 
+
     # Process xQTL data
     xqtl_data <- if (!is.null(xqtl_finemapping_obj)) readRDS(xqtl_file)[[1]][[xqtl_finemapping_obj]] else readRDS(xqtl_file)[[1]]
     xqtl_lbf_matrix <- as.data.frame(xqtl_data$susie_result_trimmed$lbf_variable)
```

Comment:
@rfeng2023 in this case you will set `xqtl_finemapping_obj = "susie_result_trimmed"` and not use this, Just `xqtl_data$lbf_variable` is good enough? Remember in a function for an R package we never want hard coded stuff like this.